tuareg.el

```diff
@@ -800,7 +800,7 @@ for the interactive mode."
           (concat extended-module-name
                   "\\(?: *\\. *" extended-module-name "\\)*"))
          (modtype-path (concat "\\(?:" extended-module-path "\\.\\)*" id))
-         (typevar "'[A-Za-z_][A-Za-z0-9_']*\\>")
+         (typevar "\\(?:'[A-Za-z_][A-Za-z0-9_']*\\|_\\)\\>")
          (typeparam (concat "[+-]?" typevar))
```

Comment:
Did not check the potential side effects but why not put the alternative on `typeparam` to avoid the regression?

Reply:
Ah, fair.  I was under the false impression that type variables in, e.g., `val f : 'a -> 'b` got fontified.